src/components/tooltip/tooltip.js

```diff
@@ -168,13 +168,13 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
           elementFocusedOnWindowBlur = false;
           return;
         }
-        parent.on('blur mouseleave touchend touchcancel', leaveHandler );
+        parent.on('blur mouseleave touchcancel', leaveHandler );
         setVisible(true);
       };
-      var leaveHandler = function () {
+      var leaveHandler = function (e) {
         var autohide = scope.hasOwnProperty('autohide') ? scope.autohide : attr.hasOwnProperty('mdAutohide');
         if (autohide || mouseActive || ($document[0].activeElement !== parent[0]) ) {
-          parent.off('blur mouseleave touchend touchcancel', leaveHandler );
+          parent.off('blur mouseleave touchcancel', leaveHandler );
```

Comment:
Not 100% sure; @rschmukler can you respond?

I think it was unnecessary and causing issues after the new gesture support, but this is exactly why the unit test above is failing.